### ISSUE TITLE
Search for contents changed.

### ISSUE THREAD

**Issue by ve3** [cadeyrn/bookmarks-organizer#224]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Some bookmark, it was there over 10 years, at first its content is about something.  
After the domain has been expired (but I don't know when) that domain was registered again by other users.  
Now that same domain has content about something else that is not related since I start bookmarked it.

For example: domain aaaa.com was bookmarked since 2002-03, its content is about cartoon 👍.  
The domain expired on 2020 and new user register that domain but content is changed to gambling 👎.  
The same domain, same page but content is completely different.

This plugin would be so powerful to check content changed.

To do this, you may need to save meta-data as you described [here](https://addons.mozilla.org/th/firefox/addon/bookmarks-organizer/reviews/1158983/).  
But to make it easier, you may have just few input to searched the content on the page. The keyword maybe grab from bookmark's keyword itself. (keyword one, keyword2, etc...)

**Additional context**
-

**Comment by cadeyrn** [comment]
Thanks for opening the issue on GitHub!

> The keyword maybe grab from bookmark's keyword itself. (keyword one, keyword2, etc...)

Unfortunately WebExtensions can't access bookmark keywords or tags.

**Comment by Illegal-Services** [comment]
I also like the idea, and would also find it useful.
@cadeyrn i think this guy @ve3 is referencing to the html meta when you request them. example:

In this example under, you can clearly and easily understand that this is the html page from Illegal Services website, if these keywords where about to be removed or changed to other strings, it would mean that the domain changed proprietary OR changed it's subject to something else...

At least that's what I've understood from this guy's message
```
<!DOCTYPE html>
<html lang="en-US">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="Illegal Services">
    <meta name="keywords" content="Illegal, Services, website">
    <meta name="author" content="IB_U_Z_Z_A_R_Dl">
</head>

<etc...>
```

**Comment by ve3** [comment]
No, I mean the bookmark itself contain Name, URL, Tags, Keyword.  
My idea is about grab the keywords from bookmark and find with string on the web page to check the contents.  
But as @cadeyrn said web extension can't access keywords so this seems to be not possible.

**Comment by Illegal-Services** [comment]
If you are a dev and wanna work on that I can probably help you with it, on parsing the html export feature.
